lib/morph-cli.rb

```diff
@@ -4,8 +4,15 @@
 
 module MorphCLI
   def self.execute(directory, development, env_config)
+    all_paths = MorphCLI.all_paths(directory)
+
+    if not all_paths.find{ |file| /scraper\.[\S]+$/ =~ file }
```

Comment:
In Ruby [use `!` instead of `not`](https://github.com/bbatsov/ruby-style-guide#bang-not-not).